src/main/java/org/apereo/openlrs/services/caliper/CaliperService.java

```diff
@@ -15,72 +15,74 @@
  */
 package org.apereo.openlrs.services.caliper;
 
-import java.util.List;
-import java.util.Map;
-
+import com.fasterxml.jackson.databind.JsonNode;
 import org.apache.log4j.Logger;
 import org.apereo.openlrs.model.OpenLRSEntity;
 import org.apereo.openlrs.model.caliper.CaliperEvent;
 import org.apereo.openlrs.model.event.EventConversionService;
 import org.apereo.openlrs.services.EventService;
-import org.imsglobal.caliper.events.Event;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.data.domain.Page;
-import org.springframework.data.domain.Pageable;
 import org.springframework.stereotype.Service;
+import java.util.List;
+import java.util.Map;
 
 /**
  * @author ggilbert
- *
+ * @author Lance E Sloan (lsloan at umich dot edu)
  */
 @Service
 public class CaliperService extends EventService {
 
-  private Logger log = Logger.getLogger(CaliperService.class);
-  
-  @Autowired
-  private EventConversionService eventConversionService;
+    private Logger log = Logger.getLogger(CaliperService.class);
+
+    @Autowired
+    private EventConversionService eventConversionService;
+
+    public void post(String organizationId, CaliperEvent caliperEvent) {
+        if (log.isDebugEnabled()) {
+            log.debug(String.format("Caliper event: %s", caliperEvent));
+        }
+
+        getTierOneStorage().save(caliperEvent);
+    }
 
-  public void post(String organizationId, CaliperEvent caliperEvent) {
-    if (log.isDebugEnabled()) {
-      log.debug(String.format("Caliper event: %s", caliperEvent));
+    public JsonNode getJsonNode(String id) {
+        OpenLRSEntity entity = getTierTwoStorage().findById(id);
+        return eventConversionService.toCaliperJson(entity);
     }
 
-    getTierOneStorage().save(caliperEvent);
+    public List<JsonNode> getJsonNodes(Map<String, String> filterMap) {
+        List<JsonNode> result = null;
+        List<OpenLRSEntity> entities = null;
+
+        if (filterMap != null && !filterMap.isEmpty()) {
+            entities = getTierTwoStorage().findWithFilters(filterMap);
+        } else {
+            entities = getTierTwoStorage().findAll();
+        }
+
+        result = eventConversionService.toCaliperJsonList(entities);
+
+        return result;
+    }
+
+    /*
```

Comment:
I'll delete them.  They were originally written by @gryglbrt, but he left them commented out because the supporting code wasn't complete.  I left them that way, but even if they're deleted, they can easily be resurrected or patterned after the xAPI support's methods.